packages/spectral/src/generators/componentManifest/templates/partials/inputs.ejs

```diff
@@ -0,0 +1,19 @@
+<% inputs.forEach((input) => { -%>
+  <%= input.key %>: {
+    inputType: "<%= input.defaultType %>",
+    <%_ if (input.collection) { -%>
+    collection: "<%= input.collection %>" as const,
+    <%_ } else { -%>
+    collection: null,
+    <%_ } -%>
+    <%_ if (input.default) { -%>
+    default: <%= input.default %>,
+    <%_ } -%>
+    <%_ if (input.required) { -%>
+    required: <%= input.required %>,
+    <%_ } -%>
+    <%_ if (input.onPremiseControlled) { -%>
```

Comment:
I'm guessing this would omit all falsy values here when if `input.onPremiseControlled === false` we'd want to include that property with a `false` value.